thehive4py/endpoints/cortex.py

```diff
@@ -27,3 +34,31 @@ def create_responder_action(
                 "responderId": responder_id,
             },
         )
+
+    def list_analyzers(self, range: Optional[str] = None) -> list[OutputAnalyzer]:
+        payload = {"range": range} if range else None
+        return self._session.make_request(
+            "GET", path="/api/connector/cortex/analyzer", json=payload
+        )
```

Comment:
As I see in the [docs](https://docs.strangebee.com/thehive/api-docs/#tag/Cortex/operation/List%20Analyzers) `range` should be a query parameter, e.g.:

```suggestion
        params = {"range": range}
        return self._session.make_request(
            "GET", path="/api/connector/cortex/analyzer", params=params
        )
```

And as far as I remember you don't really need the if-else check in this case as when a query parameter is `None` the `requests` library will just ignore it.